numpy/core/src/umath/scalarmath.c.src

```diff
@@ -587,6 +587,9 @@ static NPY_INLINE int
 @name@_ctype_absolute(@type@ a, @type@ *out)
 {
     *out = (a < 0 ? -a : a);
+    if(a < 0 && a == -a){
```

Comment:
How should I go about testing this, e.g. what test file should I add this under? I'm very unfamiliar w/ the NumPy codebase 😅

Reply:
@Micky774 yeah, not sure that using `*out` was a good idea in any case.

Many files have a corresponding test file, the same is true for scalarmath.  Since its a C file, those are in `numpy/core/tests`.   There are similar tests there (maybe even one that can be parametrized to include this one).